library/tests/fy_py_files/test_fy_py_files/property_with_generics_impl/property_with_generics_impl_fy.py

```diff
@@ -0,0 +1,31 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+"""fy
+from .greetings_t import GreetingT, SpanishGreeting
+
+
+property hello_world[GreetingT]: str using spanish_greeting:
+    property spanish_greeting[SpanishGreeting]
```

Comment:
```suggestion
    property greeting[SpanishGreeting]
```
The generic implementation class defines which greeting, but the class using the generic should be be generic since it is templated.